myst.yml

```diff
@@ -2,8 +2,9 @@
 version: 1
 project:
   title: SimPEG User Tutorials
-  # github: 
-  # description: 
+  license: CC-BY-4.0
```

Comment:
```suggestion
  licence:
   content: CC-BY-4.0
   code: MIT
```

Reply:
Just noticed that we can set two licenses through Myst. Thanks for the comment @lheagy